monai/networks/utils.py

```diff
@@ -17,8 +17,32 @@
 import torch
 import torch.nn.functional as f
 
+def one_hot(labels, num_classes, dtype=torch.float):
+    """
+    For a tensor `labels` of dimensions B1[spatial_dims], return a tensor of dimensions `BN[spatial_dims]`
+    for `num_classes` N number of classes.
+
+    Example:
+
+        For every value v = labels[b,1,h,w], the value in the result at [b,v,h,w] will be 1 and all others 0.
+        Note that this will include the background label, thus a binary mask should be treated as having 2 classes.
+    """
+    assert labels.dim() > 1, "labels should have dim of 2 or more."
```

Comment:
This `assert` will cause the classification task in CI to fail, because the label shape can be [B] without channel.
Maybe you can `unqueeze` the data if it only has batch dim?
Then check:
```py
if labels.shape[1] != 1:
    raise ValueError("labels should have a channel with length equals to one.")
```

Reply:
yes, I've noticed it too. it's fixed now